core/indigo-core/molecule/src/base_molecule.cpp

```diff
@@ -4122,6 +4128,34 @@ const int* BaseMolecule::getPyramidStereocenters(int idx) const
     return stereocenters.getPyramid(idx);
 }
 
+void BaseMolecule::setStereoFlagPosition(int frag_index, const Vec3f& pos)
+{
+    try
+    {
+        _stereo_flag_positions.insert(frag_index, pos);
+    }
+    catch (...)
```

Comment:
Why do we need such generic catch? It looks like a bad code.

Reply:
fixed